bindings/python/linkgrammar.py

```diff
@@ -51,10 +50,6 @@ def __del__(self):
             del self._obj
 
     @property
-    def version(self):
-        return clg.linkgrammar_get_version()
```

Comment:
well, thee is a dictionary version, and there is also a C library version, and the two can sometimes skew.  The goal of having a library version was to identify users with bad installs.